src/components/vm/nics/nicBody.jsx

```diff
@@ -81,7 +81,7 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
     const virtualNetwork = [{
         name: 'network',
         desc: 'Virtual network',
-        detailHeadline: _("This is the recommended config for general guest connectivity on hosts with dynamic / wireless networking configs."),
+        detailHeadline: _("This is the recommended type for general guest connectivity on hosts with dynamic / wireless networking configs."),
```

Comment:
Please revert this bit and the same config → type change below. This needlessly breaks translations, and thus is not appropriate for a stable z-stream update.